osp/utils/zulip_api.py

```diff
@@ -2,16 +2,21 @@
 
 # By default the API key file you download is named as 'download' by Zulip. You can place
 # this file one directory previous to the cuurent directory your file is in
-client = Client(config_file="download")
+
+
+def get_client():
```

Comment:
@devkapilbansal How is it going to solve the problem of `zulip_key`?
As in the CI, we need to test the zulip_api key also.
For reference see this test: https://github.com/anitab-org/open-source-programs-backend/pull/94
This test requires `download` file and the test will surely fail in CI.
Cc: @isabelcosta

Reply:
@codesankalp I think it is better to pass whole file in the client. If we are going to mock, then we can mock the whole function value as said by @isabelcosta [here](https://github.com/anitab-org/open-source-programs-backend/pull/102#issuecomment-787070652). That way we don't need Zulip Api keys in the github workflow

Reply:
I wonder if there is a cleaner way to do what I did in this PR

Reply:
I wrote the comments 3 days before [this](https://github.com/anitab-org/open-source-programs-backend/pull/102#issuecomment-787070652) @devkapilbansal,  You can ignore the above comments.
But what I want to say is that why don't we import it from `settings.py` instead of `utils.py`.

Reply:
You want to import client from settings or the whole functions. If talking about whole functions, then `zulip_api.py` provides not only functions but there examples too and it will create a whole lot of mess if we import from `settings.py`

Reply:
I suggested because in many projects the client is always defined in the `settings.py`. But if it will create a mess then leave this point. :relaxed: